packages/suite/src/components/onboarding/GlobalWebNavigation/components/LanguagePicker/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
```

Comment:
language picker is already in suite

Reply:
Not applicable now, as we have temporary build target "suite-onboarding"

Reply:
done